src/index.html

```diff
@@ -3,9 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <title>Authors Haven</title>
   </head>
   <body>
     <div id="app"></div>
   </body>
-</html>
+  <script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
Ditto!

Reply:
Another thing we can consider is using npm to load this dependency, that way development can still go on when offline.

Reply:
> Ditto!

I have implemented this feedback.

Reply:
> Another thing we can consider is using npm to load this dependency, that way development can still go on when offline.

I have implemented this too. Thank you.